resources/packs/azure/azure_ad.go

```diff
@@ -41,7 +41,7 @@ func (a *mqlAzuread) GetUsers() ([]interface{}, error) {
 	}
 	userList, err := usersClient.Users().Get()
 	if err != nil {
-		return nil, err
+		return nil, msgraphclient.TransformODataError(err)
```

Comment:
Should azure really depend on the msgraph package? Is there a odataerror package in the azure sdk

Reply:
I can move the error transformion out. The msgraph package has the odataerror in it, we use the `msgraphclient` for users,domains, etc.

This probably brings up a good point of doing the same thing twice: via `ms365` and via `azure` providers. Should we consider dropping one of those?